app/models/concerns/cms/addon/sns_poster.rb

```diff
@@ -0,0 +1,179 @@
+module Cms::Addon
+  module SnsPoster
+    extend ActiveSupport::Concern
+    include Cms::Content
+    extend SS::Addon
+
+    included do
+
+      field :twitter_auto_post,  type: String, default: 'expired'
+      field :facebook_auto_post, type: String, default: 'expired'
+      field :sns_auto_delete,    type: String, default: 'expired'
+      field :twitter_user_id,    type: String
+      field :twitter_post_id,    type: String
+      field :facebook_user_id,   type: String
+      field :facebook_post_id,   type: String
+
+      permit_params :facebook_auto_post,
+                    :twitter_auto_post,
+                    :sns_auto_delete,
+                    :twitter_post_id,
+                    :twitter_user_id,
+                    :facebook_user_id,
+                    :facebook_post_id
+
+      after_generate_file { post_sns }
+      after_remove_file { delete_sns }
+    end
+    
+    def sns_poster_facebook_options
+      definition_state
+    end
+
+    def sns_poster_facebook_options_ja
+      I18n.t("views.options.state.#{facebook_auto_post}")
+    end
+
+    def sns_poster_twitter_options
+      definition_state
+    end
+
+    def sns_poster_twitter_options_ja
+      I18n.t("views.options.state.#{twitter_auto_post}")
+    end
+
+    def sns_poster_delete_options
+      definition_state
+    end
+
+    def sns_poster_delete_options_ja
+      I18n.t("views.options.state.#{sns_auto_delete}")
+    end
+
+    def access_token_facebook(snskeys)
+      access_token = snskeys["access_token_facebook"]
+      graph = Koala::Facebook::API.new(access_token)
+    end
+
+    def message_format(html)
+      html = ActionController::Base.helpers.strip_tags(html)
+      html = ActionController::Base.helpers.truncate(html, :length=> 253)
+    end
+
+    def use_twitter_post?
+      twitter_auto_post == "active"
+    end
+
+    def use_facebook_post?
+      facebook_auto_post == "active"
+    end
+
+    def twitter_url
+      "https://twitter.com/#{twitter_user_id}/status/#{twitter_post_id}" if use_twitter_post? && twitter_user_id.present? && twitter_post_id.present?
+    end
+
+    def facebook_url
+      "https://www.facebook.com/#{facebook_user_id}/posts/#{facebook_post_id}" if use_facebook_post? && facebook_user_id.present? && facebook_post_id.present?
```

Comment:
__[rubocop]__
* Line is too long. [158/130] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->